grazer/src/main/java/edu/asu/diging/grazer/core/domain/ITransformationFile.java

```diff
@@ -0,0 +1,27 @@
+package edu.asu.diging.grazer.core.domain;
+
+import java.util.HashMap;
+import java.util.List;
+
+import org.springframework.web.multipart.MultipartFile;
+
+/**
+ * 
+ * This class stores the file names and file data of one pattern file and its 
+ * corresponding transformation file. 
+ * 
+ * @author mshah18
+ *
+ */
+public interface ITransformationFile {
+
+    
+    MultipartFile getTransformationFile();
```

Comment:
your domain classes should not depend on a spring framework web class, unless there is no good way around it.